bankgreen/settings.py

```diff
@@ -68,11 +68,13 @@
     "django_admin_listfilter_dropdown",
     "django_filters",
     "django_json_widget",
+    "django_prometheus",
```

Comment:
Shouldn't this dependency be added to `requirements.txt`?

Reply:
Yes, my bad.